content/v2.0/application/cli-commands.md

```diff
@@ -0,0 +1,181 @@
+---
+title: CLI commands
+order: 70
+---
+
+Hanami provides a command line interface powered by the [hanami-cli](https://github.com/hanami/cli) gem, with helpful commands for generating a new application, starting a console, starting a development server, displaying routes and more.
+
+## Gem commands
+
+After an initial install via `gem install hanami`, hanami offers two commands:
+
+```ruby
+hanami --help
+
+Commands:
+  hanami new APP         # Generate a new Hanami app
+  hanami version         # Hanami version
+```
+
+### hanami new
+
+Generates a Hanami application with the given APP name, in a new directory from the current location.
+
+```shell
+hanami new bookshelf # generates a new Bookshelf application in ./bookshelf
+hanami new my_app # generates a new MyApp application in ./my_application
+```
+
+
+### hanami version
+
+Displays the Hanami version of the currently installed gem.
+
+```shell
+hanami version
+v2.0.0
+```
+
+
+## Project commands
+
+When executed from the root of a Hanami project, hanami offers a different set of commands.
```

Comment:
```suggestion
When executed within an Hanami project, hanami offers a different set of commands.
```

(I made a fix last week that ensures `hanami` works to offer app commands even when invoked within deeper app directories)